### PR TITLE
API: Add get intervention segment. RD-27228

### PR DESCRIPTION
**Pull request by alexeykondratenko**

### CODE REVIEW

**Comment by ylecuyer** [comment]
Could you add screen of the part that were added to the doc?

**Comment by alexeykondratenko** [comment]
> Could you add screen of the part that were added to the doc?

int_segm

**Comment by alexeykondratenko** [comment]
> Could you add the swagger part please?

swagger_1
swagger_2

File: specs/engage-digital_openapi3.yaml [hunk 1]
```diff
@@ -8524,6 +8549,52 @@ components:
...
+        updated_at:
+          format: date-time
+          type: string
+        user_id:
+          type: string
+        auto_summary:
+          type: string
+          description: |-
+            Auto summary of the intervention segment.
+            Only present when the conversation summary extension is enabled or with RingCX.
```
Comment (ylecuyer) [inline specs/engage-digital_openapi3.yaml:8582]:
```suggestion
            Only present with RingCX.

```

File: specs/engage-digital_openapi3.yaml [hunk 2]
```diff
@@ -8524,6 +8549,52 @@ components:
...
+        agent_summary:
+          type: string
+          description: |-
+            Agent's summary of the intervention segment.
+            Only present when the conversation summary extension is enabled or with RingCX.
```
Comment (ylecuyer) [inline specs/engage-digital_openapi3.yaml:8587]:
```suggestion
            Only present with RingCX.

```

File: specs/engage-digital_openapi3.yaml [hunk 3]
```diff
@@ -8524,6 +8549,52 @@ components:
...
+        summary_edited:
+          type: boolean
+          description: |-
+            Indicates if the summary was edited by the agent.
+            Only present when the conversation summary extension is enabled or with RingCX.
```
Comment (ylecuyer) [inline specs/engage-digital_openapi3.yaml:8592]:
```suggestion
            Only present with RingCX.

```

File: specs/engage-digital_openapi3.yaml [hunk 4]
```diff
@@ -8524,6 +8549,52 @@ components:
...
+        generate_auto_summary:
+          type: boolean
+          description: |-
+            Indicates if the intervention segment supports summary generation.
+            Only present when the conversation summary extension is enabled or with RingCX.
```
Comment (ylecuyer) [inline specs/engage-digital_openapi3.yaml:8597]:
```suggestion
            Only present with RingCX.

```

Comment (alexeykondratenko) [reply specs/engage-digital_openapi3.yaml]: Done

File: specs/engage-digital_openapi3.yaml (hunk 3 above)
Comment (alexeykondratenko) [reply specs/engage-digital_openapi3.yaml]: Done

File: specs/engage-digital_openapi3.yaml (hunk 2 above)
Comment (alexeykondratenko) [reply specs/engage-digital_openapi3.yaml]: Done

File: specs/engage-digital_openapi3.yaml (hunk 1 above)
Comment (alexeykondratenko) [reply specs/engage-digital_openapi3.yaml]: Done

**Comment by ylecuyer** [comment]
LGTM

**Review by ylecuyer — APPROVED** [review]

**Pull request merged**